jekyll/_cci2/runner-installation.adoc

```diff
@@ -77,21 +81,39 @@ First, set one of these variables as appropriate for for your installation targe
 | `platform=darwin/arm64`
 |===
 
-Run the following steps to download, verify and install the binary.
+Next, set the `circleci-launch-agent` version. On cloud, we select the latest version by
+querying for our releases for it and on server, we must select the version
```

Comment:
This sentence is confusing me... "On cloud, we query our releases for the latest version..." maybe?

Reply:
Is it more direct to say something like "Runners on cloud auto-update to the latest supported versions.  For Server, specific runner versions are validated for interoperability and runners do not auto-update"

That gets around the use of "we", if nothing else :)

Reply:
I'll add this change